rundeckapp/src/test/groovy/rundeck/services/ProjectServiceSpec.groovy

```diff
@@ -344,4 +357,41 @@ class ProjectServiceSpec extends Specification {
             0 * fwk.getFrameworkProjectMgr()
             !result.success
     }
+
+    def "import project archive"() {
+        setup:
+        defineBeans {
+            webhookImporter(WebhooksProjectImporter)
+        }
+        def project = Mock(IRundeckProject) {
+            getName() >> 'importtest'
+        }
+        def framework = Mock(Framework) {
+            getFrameworkProjectsBaseDir() >> { File.createTempDir() }
+        }
+        def authCtx = Mock(UserAndRolesAuthContext) {
+            getUsername() >> {"user"}
+            getRoles() >> {["admin"] as Set}
+        }
+        service.scheduledExecutionService = Mock(ScheduledExecutionService) {
+            loadJobs(_,_,_,_,_,_) >> { [] }
+            issueJobChangeEvent(_) >> {}
+        }
+        service.logFileStorageService = Mock(LogFileStorageService) {
+            getFileForExecutionFiletype(_,_,_,_) >> { File.createTempFile("import","import") }
+        }
+        ProjectArchiveParams rq = new ProjectArchiveParams()
+        rq.project = "importtest"
+        rq.importConfig = true
+        rq.importACL = true
+        rq.importScm = true
+        rq.importWebhooks=true
+
+        when:
+        def result = service.importToProject(project,framework,authCtx, getClass().getClassLoader().getResourceAsStream("test-rdproject.jar"),rq)
+        println result
+
+        then:
+        result
```

Comment:
can this test verify the webhooks were successfully imported?

Reply:
This test is designed to check if the webhook feature is enabled and a project archive doesn't have any webhook definitions in it, that the import does not throw the error described in the issue.

The webhook import tests are in a different test.

Reply:
ok, maybe indicate that in the test name